js/background.js

```diff
@@ -559,7 +559,7 @@ var count_cache = (function(count_map) {
     var cache = count_map;
 
     // Update custom filter count stored in localStorage
-    var _updateCustomFilterCount = function() {
+    function _updateCustomFilterCount() {
```

Comment:
@kpeckett Yes, it's right as part of the switching from function expressions to function declarations. The scope of the function hasn't been changed.